tools/gulp/tasks/publish.ts

```diff
@@ -42,19 +42,26 @@ task(':publish:whoami', execTask('npm', ['whoami'], {
 task(':publish:logout', execTask('npm', ['logout']));
 
 task(':publish', async () => {
-  const label = argv['tag'];
+  const tag = argv['tag'];
   const version = buildConfig.projectVersion;
   const currentDir = process.cwd();
 
   console.log('');
-  if (!label) {
-    console.log(grey('> You can use a label with --tag=labelName.\n'));
-    console.log(green(`Publishing version "${version}" using the latest tag...`));
+  if (!tag) {
+    console.log(grey('> You can specify the tag by passing --tag=labelName.\n'));
+    console.log(green(`Publishing version "${version}" within the latest tag...`));
   } else {
-    console.log(yellow(`Publishing version "${version}" using the ${label} tag...`));
+    console.log(yellow(`Publishing version "${version}" within the ${tag} tag...`));
   }
   console.log('');
 
+
+  if (version.includes('beta') && (!tag || tag === 'latest')) {
```

Comment:
Maybe we should do the same for `alpha` and `rc`?

Reply:
Thought the same about `rc`, but wasn't sure about that one, because I think it can also happen that RCs are being relased into the `latest` tag.

Reply:
Nah, we should never have `latest` pointing to an "rc" again, so that would be good too. I don't believe we'll ever have an "alpha" release again, but it wouldn't hurt to put it in there anyway.

Reply:
Done.